package.json

```diff
@@ -42,11 +42,14 @@
     "@types/react": "^16.9.38",
     "@types/styled-components": "^5.1.0",
     "csstype": "^3.0.0-beta.4",
+    "moment": "^2.29.1",
```

Comment:
Damn, I decided to go with `dayjs` package which provides the same API as `moment` but is way smaller. Is the usage of `moment` hard requirement of the `react-dates`? Or can we replace it?